index.js

```diff
@@ -125,5 +136,11 @@ const getColor = (percentile) => {
 sense.Leds.lowLight = true
 sense.Leds.clear()
 sense.Leds.flipH()
-run()
 
+run();
+
+// If we're event driven, we only ever do something if an event drops on us
+if (eventDriven) {
+  // Initiate
+  frontInstance.registerEvents({ port: 80 }, run);
```

Comment:
Ah, I see. So the event doesn't contain a data payload, but triggers the check to see what changed. Is there a way to change it so that it never needs to fetch anything, and just maintains local state after bootstrapping? I don't think we want that for notification devices, but it would be more efficient if we were actually maintaining a fleet of them.